web/components/atoms/ProgressBar.tsx

```diff
@@ -0,0 +1,16 @@
+interface Props {
+	color?: string;
+	/** A number between 0 and 100 indicating the progress percentage */
+	progress: number;
+}
+
+export default function ProgressBar({
+	color,
+	progress
+}: Props) {
+	return (
+		<div className="h-[6px] relative rounded-full">
```

Comment:
Can you also add the gray on the other side of the progress bar? I think you can just add `bg-[color]` to this div, and you can add the gray color to `web/tailwind.config.js` too so that it can be easily reused and updated!